### ISSUE TITLE
Handle "NoRegion"/"Global" charges

### ISSUE THREAD

**Issue by vsmart** [thegreenwebfoundation/green-cost-explorer#15]
Some charges have no region info attached to them or are global. 

How should these be handled? Grey by default, a third "neutral" category, or disregarded entirely?

**Comment by mrchrisadams** [comment]
Hmm… I think for some services, like S3 or Route53, you can't choose for them to green, as they work across regions.

If we want to make it possible for people to have 100% green and 0% grey, we'd need a neutral.

On the other hand, they _still_ would be grey, and it would highlight that AWS needs to be _entirely_ green, not just have _some_ sustainable regions.

Climate Emergency and all that.

I honestly don't know for sure, but I'm leaning towards grey, to highlight you need systemic change.

Lets chat in person then make  record the decision with architectural decision record, as this will likely have implications in how we present info at all levels.

https://adr.github.io/

**Comment by conan** [comment]
My tax is all reported as being grey, presumably because the entities that collect it are located in non-green locations; it's not clear to me how tax relates to emissions however.

**Comment by mrchrisadams** [comment]
Yeah, that sounds like we should probably just omit it. Thanks for flagging it :)
